tests/codeception/acceptance/Contrib/XmlsitemapCest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * Test that soe_profile_xmlsitemap_link_alter is working.
+ */
+class XmlsitemapCest {
+
+  /**
+   * Test that xmlsitemap is having 403 and 404 pages removed.
+   */
+  public function testSoeSitemapLinkAlter(AcceptanceTester $I) {
+    $I->logInWithRole('administrator');
+    $I->amOnPage("/admin/config/search/xmlsitemap/rebuild");
+    $I->click('Save configuration');
+    $I->runDrush('cr');
+    $I->amOnPage("/sitemap.xml");
+    $host = \Drupal::request()->getSchemeAndHttpHost();
+    $soe_profile_403_page = \Drupal::config('system.site')->get('page.403');
+    $alias_403 = \Drupal::service('path.alias_manager')->getAliasByPath($soe_profile_403_page);
+    $soe_profile_404_page = \Drupal::config('system.site')->get('page.404');
+    $alias_404 = \Drupal::service('path.alias_manager')->getAliasByPath($soe_profile_404_page);
```

Comment:
Normally when i use services or configs like this, i'll set a variable for that service. Then drupal doesn't have to go out and build the service/config object again. 

But its just semantics. Honestly i wasn't sure if `\Drupal` was available in these. Its pretty handy.